alluxiofs/core.py

```diff
@@ -145,45 +160,75 @@ def _strip_protocol(path):
         self.error_metrics = AlluxioErrorMetrics()
 
     def unstrip_protocol(self, path):
-        path = self._strip_alluxio_protocol(path)
         if self.fs:
-            # avoid adding Alluxio protocol to the full ufs url
+            # avoid adding Alluxiofs protocol to the full ufs url
             return self.fs.unstrip_protocol(path)
         return path
 
     def get_error_metrics(self):
         return self.error_metrics.get_metrics()
 
+    def _translate_alluxio_info_to_fsspec_info(self, file_status, detail):
+        if detail:
+            return {
+                "name": self._strip_protocol(file_status.ufs_path),
+                "type": file_status.type,
+                "size": file_status.length
+                if file_status.type == "file"
+                else None,
+                "last_modification_time_ms": getattr(
+                    file_status, "last_modification_time_ms", None
+                ),
+            }
+        else:
+            return self._strip_protocol(file_status.ufs_path)
+
     def fallback_handler(alluxio_impl):
         @wraps(alluxio_impl)
-        def fallback_wrapper(self, path, *args, **kwargs):
-            path = self._strip_alluxio_protocol(path)
+        def fallback_wrapper(self, *args, **kwargs):
+            signature = inspect.signature(alluxio_impl)
+            bound_args = signature.bind(self, *args, **kwargs)
+            bound_args.apply_defaults()
+            for param in ["path", "path1", "path2", "lpath", "rpath"]:
```

Comment:
Could you document why the params are `["path", "path1", "path2", "lpath", "rpath"]`?

Reply:
yeah, np